### PR TITLE
Updated script and instructions to work in Windows

### PR DESCRIPTION
**Pull request by jmoreira-valory**

### CODE REVIEW

File: run_service.sh [hunk 1]
```diff
@@ -10,6 +10,12 @@ then
 fi
 
 # Check dependencies
+if command -v python3 >/dev/null 2>&1; then
+    PYTHON_CMD="python3"
+else
+    PYTHON_CMD="python"
+fi
```
Comment (Adamantios) [inline run_service.sh:17]: Let's perform the check for `python` too.

**Review by Adamantios — APPROVED** [review]

File: run_service.sh (hunk 1 above)
Comment (jmoreira-valory) [reply run_service.sh]: Addressed

File: run_service.sh [hunk 2]
```diff
@@ -142,22 +158,22 @@ then
     agent_balance=0
     operator_balance=0
     suggested_amount=50000000000000000
-    until [[ $(python3 -c "print($agent_balance > ($suggested_amount-1))") == "True" && $(python3 -c "print($operator_balance > ($suggested_amount-1))") == "True" ]];
+    until [[ $($PYTHON_CMD -c "print($agent_balance > ($suggested_amount-1))") == "True" && $($PYTHON_CMD -c "print($operator_balance > ($suggested_amount-1))") == "True" ]];
     do
-        echo "Agent instance's balance: $agent_balance WEI."
-        echo "Operator's balance: $operator_balance WEI."
+        echo "Agent address: $agent_address. Balance: $agent_balance WEI."
+        echo "Operator address: $operator_address. Balance: $operator_balance WEI."
```
[the remark this answers is not in the document]
Comment (jmoreira-valory) [reply run_service.sh:164]: Fixed.

**Review by Adamantios — APPROVED** [review]

**Comment by dagacha** [comment]
LGTM

**Pull request merged**